apps/bro-spinn/build.gradle.kts

```diff
@@ -1,12 +1,17 @@
+val jacksonVersion: String by project
 val ktorVersion: String by project
 val spinnInntektsmeldingKontraktVersion: String by project
 
 dependencies {
     implementation(project(":felles-auth"))
-    implementation("io.ktor:ktor-client-core:$ktorVersion")
+    implementation("com.fasterxml.jackson.dataformat:jackson-dataformat-xml:$jacksonVersion")
+    implementation("com.fasterxml.jackson.datatype:jackson-datatype-jsr310:$jacksonVersion")
+    implementation("com.fasterxml.jackson.module:jackson-module-kotlin:$jacksonVersion")
```

Comment:
Disse pakkene hentes i dag via rr-pakken. Legger disse til som en direkte avhengighet for å ha mer kontroll over egne avhengigheter.